test-suite/misc/PStringExtraction.out

```diff
@@ -0,0 +1,71 @@
+File "./misc/PStringExtraction.v", line 25, characters 0-23:
+Warning: The following axiom must be realized in the extracted code:
+ int.
+ [extraction-axiom-to-realize,extraction,default]
+
+type comparison =
+| Eq
+| Lt
+| Gt
+
+type int (* AXIOM TO BE REALIZED *)
+
+type char63 = int
+
+(** val make : int -> char63 -> Pstring.t **)
+
+let make = Pstring.make
+
+(** val length : Pstring.t -> int **)
+
+let length = Pstring.length
+
+(** val sub : Pstring.t -> int -> int -> Pstring.t **)
+
+let sub = Pstring.sub
+
+(** val cat : Pstring.t -> Pstring.t -> Pstring.t **)
+
+let cat = Pstring.cat
+
+(** val compare : Pstring.t -> Pstring.t -> comparison **)
+
+let compare = (fun x y -> let c = Pstring.compare x y in if c = 0 then Eq else if c < 0 then Lt else Gt)
+
+module Char63Notations =
```

Comment:
Is it possible to make it so that this module `Char63Notations` is never extracted?

Reply:
That's because `char63_wrapper` was defined in the module, I moved it to the outside and that did the trick.